docket/api.py

```diff
@@ -1,20 +1,28 @@
 from flask import Flask, Blueprint, Response
-from flask_restful import Resource, Api
+from flask_restful import Api
 
-from resources.pcap import PcapApi, PcapUri, RawQuery
-from resources.stats import Stats
+from resources.query import QueryRequest, ApiRequest, RawRequest
+
+from config import Config
 
 # Declare the blueprint
 api_bp = Blueprint('api', __name__)
-api = Api(api_bp)
+api    = Api(api_bp)
 
 # Add resources
-api.add_resource(PcapApi, '/api/')
-api.add_resource(PcapUri, '/pcap/<path:path>')
-api.add_resource(RawQuery, '/query' )
-api.add_resource(Stats, '/stats/', '/stats/<string:sensors>', endpoint='stats')
+WEB_ROOT = Config.get('WEB_ROOT', '/')
+# consider 'login_required' for queries
+api.add_resource(RawRequest, WEB_ROOT+'raw/<string:query>' )     # ex: /raw/host+1.2.3.4+port+80
```

Comment:
Originally the `RawQuery` was to provide backwards compatibility with `stenoread`. This would allow the existing CLI client to query an array of stenographer instances. This requires changes in the `query.py` file (also noted), but it must be rooted at `/query` in the URI to work.

Reply:
Good to know.

Reply:
So... stenoread is going to expect packet data.  It will currently receive a block of JSON indicating the query was queued. I can make uwsgi wait for completion, read and return the file data... but I think spending time on a web UI is ... better.